From b509d9189c0a9fbfa215242d6b585f404ae94adf Mon Sep 17 00:00:00 2001
From: Xing Zheng <zhengxing@rock-chips.com>
Date: Mon, 28 May 2018 16:57:03 +0800
Subject: [PATCH] arm64: dts: rockchip: add loopback node for RK3308 EVB

RK3308 EVB V10: using ADC7 and ADC8 (grp==3) for loopback
RK3308 EVB V11: using ADC1 and ADC2 (grp==0) for loopback

Change-Id: I88a24bddacd25ba54878e97dec8b08b4f38875a2
Signed-off-by: Xing Zheng <zhengxing@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi | 1 +
 arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi b/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
index f4bceb77b8b4..8b5e7dfcc8a9 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-v10.dtsi
@@ -241,6 +241,7 @@
 
 	#sound-dai-cells = <0>;
 
+	rockchip,loopback-grp = <3>;
 	hp-ctl-gpios = <&gpio0 RK_PA1 GPIO_ACTIVE_HIGH>;
 	spk-ctl-gpios = <&gpio0 RK_PA5 GPIO_ACTIVE_HIGH>;
 };
diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi b/arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi
index d54f53907e84..43f89939e713 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-v11.dtsi
@@ -247,6 +247,7 @@
 
 	#sound-dai-cells = <0>;
 
+	rockchip,loopback-grp = <0>;
 	hp-ctl-gpios = <&gpio0 RK_PA1 GPIO_ACTIVE_HIGH>;
 	spk-ctl-gpios = <&gpio0 RK_PA5 GPIO_ACTIVE_HIGH>;
 };
-- 
2.35.3

